packages/api/src/comet-brevo-module.ts

```diff
@@ -0,0 +1,12 @@
+import { DynamicModule, Global, Module } from "@nestjs/common";
+
+@Global()
+@Module({})
+export class CometBrevoModule {
```

Comment:
I'd drop the Comet prefix, just call it BrevoModule

Reply:
I dropped the prefix in [38a792a](https://github.com/vivid-planet/comet-brevo-module/pull/3/commits/38a792aff39ac7019ec5c9155c85e173e3419c8b)